ex6/ex6.c

```diff
@@ -21,6 +22,21 @@ and `clock_gettime()` should work just fine.
 int main()
 {
     // Your code here
+    struct timespec start, end;
+    __uint64_t count = 0;
+    __uint64_t av = 0;
+    __uint64_t diff;
+    printf("Running\n");
+    while( count != number_iter){
+    clock_gettime(CLOCK_MONOTONIC, &start);
+        fprintf( stdout, "" );
+    clock_gettime(CLOCK_MONOTONIC, &end);
     
+    diff = BILLION * (end.tv_sec - start.tv_sec) + end.tv_nsec - start.tv_nsec;
+    count++;
+    av = ((av * (count - 1)) + diff ) / count;
+        
+    }
+    printf("%d \n", av);
```

Comment:
My compiler wants you to change the %d to something that will match the __uint64_t data type. I believe this would be %PRIu64.

Reply:
This is running fine for me, but I've updated it to `%lld` since this site says it works on all compilers 

https://www.viva64.com/en/k/0046/